sa/_db/migrations/20150904121736_CertificateStatusNotNullManyFields.sql

```diff
@@ -0,0 +1,23 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `certificateStatus` MODIFY `subscriberApproved` tinyint(1) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `status` varchar(255) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `ocspLastUpdated` datetime NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `revokedDate` datetime NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `revokedReason` int(11) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `lastExpirationNagSent` datetime NOT NULL;
```

Comment:
This is probably NULL before we send the first expiration nag, unless we code it to be the epoch or somesuch.